src/app/modules/shared-components/components/progress-level/progress-level.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'alg-progress-level',
+  templateUrl: './progress-level.component.html',
+  styleUrls: ['./progress-level.component.scss']
+})
+export class ProgressLevelComponent {
+
+  @Input() size = 12;
+  @Input() theme : 'default' | 'revert' = 'default';
+  @Input() filled = 0;
```

Comment:
This attribute name is not clear... from what I understand it is just the number of dots filled... ok so it may just be called "value". And if you call "total" -> "maximum", you have the same kind of properties as a numeric slider... which is actually quite close to what this widget is.

Reply:
fiixed